microsoft-edge/toc.yml

```diff
@@ -1029,89 +1046,167 @@
+
+                - name: REST API Reference for Microsoft Edge Add-ons
+                  href: extensions-chromium/publish/api/addons-api-reference.md
+
+# -------------------------------------
+        - name: Manage and maintain an extension
+          items:
+            - name: Update a Microsoft Edge extension
+              href: extensions-chromium/publish/update-extension.md
+
+            - name: Automatically update an extension in Microsoft Edge  # todo: update top-of-article title
+              href: extensions-chromium/publish/auto-update.md  # moved from /enterprise/
+
             - name: View Microsoft Edge extensions analytics dashboard
               href: extensions-chromium/publish/extensions-analytics.md
 
             - name: Reply to user reviews
               href: extensions-chromium/publish/reply-user-reviews.md
-              displayName: Respond to user reviews
+              displayName: Reply to user reviews about your Microsoft Edge extension, Respond to user reviews  # top-of-page title, alt title word  # todo: make top-of-title identical to toc title, clarify in first sentence
 
-            - name: Manage account settings
-              href: extensions-chromium/publish/manage-settings.md
+#           - name: Connect with your Customer  # how you can connect with your customers
+#             href:
 
-            - name: Contact Microsoft Edge extensions support
-              href: extensions-chromium/publish/contact-extensions-team.md
-              displayName: feedback, log issues, enter issues, report issues, reporting issues
+#           - name: User Ratings  # how User Ratings work, eg how region-wide users can see the review & ratings
+#             href:
 
-            - name: Policies for Microsoft Edge Add-ons
-              items:
-                - name: Content Security Policy (CSP)
-                  href: extensions-chromium/store-policies/csp.md
+#           - name: Review Process  # the review process for an extension that's submitted to add-ons site
+#             href:
 
-                - name: App Developer Agreement Addendum for Microsoft Edge program users
-                  href: extensions-chromium/store-policies/ada-addendum.md
+#           - name: Retire your Extension  # how to uninstall or retire an extension
+#             href:
 
-                - name: Developer policies for the Microsoft Edge Add-ons store
-                  href: extensions-chromium/store-policies/developer-policies.md
+# -------------------------------------
+        - name: Best practices
+          items:
+#           - name: Checklist for publishing an extension
+#             href:
 
-            - name: Microsoft Edge Add-ons API
-              items:
-                - name: Using the Microsoft Edge Add-ons REST API
-                  href: extensions-chromium/publish/api/using-addons-api.md
+            - name: Best practices for extensions
```

Comment:
disparity:
toc title: "practices"
top-of-article title: "Practices"

i will globally make case consistent
![image](https://github.com/user-attachments/assets/ee75034f-9efc-4a18-9183-7f3683b92d60)